daffodil-io/src/main/scala/org/apache/daffodil/io/DataOutputStream.scala

```diff
@@ -174,7 +174,7 @@ trait DataOutputStream extends DataStreamCommon
    * It is a usage error if bitLengthFrom1 is not greater than or equal to 1.
    *
    */
-  def putByteArray(ba: Array[Byte], bitLengthFrom1: Int, finfo: FormatInfo): Boolean
+  def putByteArray(ba: Array[Byte], bitLengthFrom1: Int, finfo: FormatInfo, ignoreByteOrder: Boolean = false): Boolean
```

Comment:
ignoreByteOrder begs for scaladoc. Otherwise,.. how the heck can that be a sensible thing to do?